packages/create-nostalgie-app/createNostalgieApp.js

```diff
@@ -45,6 +45,12 @@ function init() {
             boolean: true,
             description: 'Force scaffolding even if the target directory is not empty',
           },
+          template: {
+            string: true,
+            default: DEFAULT_TEMPLATE_SPEC,
```

Comment:
Let's leave this without default and layer on the default in the handler. The reason for this is that setting this as the default will put the specifier in the `--help` text. Since it is long and in a relatively obscure format, I think it will be: 1) visually unappealing; and 2) of little added value.

I think default values are helpful for those where there is some ambiguity about what will be used by default. In this case, I suspect a user will _expect_ a suitable template to be used by default and is likely to be _MORE_ confused if they see some obscure degit syntax (😅).